### PR TITLE
fix is_bluesky_specfile and remove voigt model from fitting routine i…

### PR DESCRIPTION
**Pull request by strempfer**
…n diffraction

### CODE REVIEW

File: polartools/load_data.py [hunk 1]
```diff
@@ -232,12 +232,12 @@ def is_Bluesky_specfile(spec_file, folder=''):
...
     """
-    if isinstance(spec_file, str):
-        path = join(folder, spec_file)
+    if isinstance(source, str):
+        path = join(folder, source)
         source = SpecDataFile(path)
 
     for comment in source.headers[0].comments:
-        if 'Bluesky' in comment:
+        if "Bluesky" in comment:
```
Comment (prjemian) [inline polartools/load_data.py:240]: A stronger verification would check only the first comment.

```python
    if len(source.headers[0].comments):
        return source.headers[0].comments[0].startswith("Bluesky")
```

**Review by gfabbris — APPROVED** [review]

File: polartools/diffraction.py [hunk 2]
```diff
@@ -63,7 +63,6 @@ def fit_peak(xdata, ydata, model="Gaussian", output=False):
     models = {
         "Gaussian": GaussianModel(),
         "Lorentzian": LorentzianModel(),
-        "Voigt": VoigtModel(),
```
Comment (gfabbris) [inline polartools/diffraction.py:66]: You can remove the VoigtModel from line 15, see LGTM.

**Comment by strempfer** [comment]
Anything else? Otherwise I merge to main.

**Comment by prjemian** [comment]
Comment on my suggestion about `if "Bluesky" in comment` above?

File: polartools/load_data.py (hunk 1 above)
Comment (strempfer) [reply polartools/load_data.py:240]: > A stronger verification would check only the first comment.
> 
> ```python
>     if len(source.headers[0].comments):
>         return source.headers[0].comments[0].startswith("Bluesky")
> ```

Very elegant. I implemented it and it seems to work.

**Comment by gfabbris** [comment]
I'll add some extra tests to this function, so don't merge it for now.

**Comment by gfabbris** [comment]
I had to change a bit the `is_Bluesky_specfile` to cover the cases when `SPEC` generates an empty header:
```python
if len(source.headers[0].comments):
    return source.headers[0].comments[0].startswith("Bluesky")
else:
    return False
```

**Pull request merged**